UM-USElab/gradecraft-development
# 3962 Update Google Client Library Gem

### Status
**READY**

### Description
The Google Calendars library 0.10 is being deprecated. Update to ver. 0.11

### Related PRs
N/A

### Todos

### Deploy Notes
N/A

### Gem dependencies
- UPDATED [google-api-client](https://github.com/google/google-api-ruby-client)

### Migrations
NO

### Steps to Test or Reproduce
As an Instructor and Student, verify that you can still add events and assignments to your affiliated Google Calendar via the application. A user should be able to add individual assignments and events as well as batch (Add all...) assignments and events to the associated Google Calendar.

### Impacted Areas in Application
List general components of the application that this PR will affect:

* Events, Assignments

======================
Closes #3962
